build.gradle

```diff
@@ -698,8 +698,8 @@ dependencies {
 
     // Only osx-x86_64, osx-aarch_64, linux-x86_64, linux-aarch_64, windows-x86_64 are available
     if (osdetector.classifier in ["osx-x86_64", "osx-aarch_64", "linux-x86_64", "linux-aarch_64", "windows-x86_64"]) {
-        testImplementation "io.netty:netty-tcnative-classes:2.0.61.Final"
-        testImplementation "io.netty:netty-tcnative-boringssl-static:2.0.61.Final:${osdetector.classifier}"
+        testImplementation "io.netty:netty-tcnative-classes:2.0.66.Final"
```

Comment:
Hm... these changes seem to be unrelated?

Reply:
These deps are related, as ZK includes `netty-tcnative-boringssl-static` and `netty-tcnative-classes`:
https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper/3.9.3